From 25bc4d4647a416066c3fa866d6180d16933fd219 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:42 -0400
Subject: [redhat] kabi: Adding symbol blk_queue_bounce (mm/bounce.c)

Message-id: <2bdde2c390588d8531f7fd2ab1b699efb573a403.1410510773.git.poros@redhat.com>
Patchwork-id: 95926
O-Subject: [PATCH RHEL7.1 042/233] kabi: Adding symbol blk_queue_bounce (mm/bounce.c)
Bugzilla: 1062361
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to process bounce buffer.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_queue_bounce
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_queue_bounce
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_queue_bounce

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_bounce b/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_bounce
new file mode 100644
index 0000000..86e46fe
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_bounce
@@ -0,0 +1,2 @@
+#1-
+0xadb55f7f blk_queue_bounce vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_queue_bounce b/redhat/kabi/kabi-module/kabi_s390x/blk_queue_bounce
new file mode 100644
index 0000000..ab71f77
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_queue_bounce
@@ -0,0 +1,2 @@
+#1-
+0xeeead523 blk_queue_bounce vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_bounce b/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_bounce
new file mode 100644
index 0000000..c6d9716
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_bounce
@@ -0,0 +1,2 @@
+#1-
+0x4061b663 blk_queue_bounce vmlinux EXPORT_SYMBOL
-- 
1.7.1